src/index.js

```diff
@@ -2,35 +2,95 @@
 import _ from "lodash";
 import "./style.css";
 
-const todoArr = [
-  {
-    description: "Take a walk",
-    completed: false,
-    index: 0,
-  },
-  {
-    description: "Go shopping",
-    completed: false,
-    index: 1,
-  },
-];
-
-function component() {
-  const element = document.createElement("div");
-  element.classList.add = "list-items";
-
-  todoArr.forEach((item, index) => {
-    if (item.index === index) {
-      element.innerHTML += `<div class="list-items border">
-            <input type="checkbox" class="start"></i> 
-            <p> ${item.description}</p>
-            <i class="bi bi-three-dots-vertical end"></i>
-        
-            </div>`;
+import displayTodo from "./modules/render.js";
+
+export const listBox = document.getElementById("listBox");
+const form = document.getElementById("todoform");
+const formInput = document.getElementById("text");
+
+export const todosArr = JSON.parse(localStorage.getItem("todo")) || [];
```

Comment:
- [ ] Hi. Apart from your add todo function, the rest of them throw the same error
<img width="756" alt="image" src="https://user-images.githubusercontent.com/37901469/194517546-0b9e5ee9-1bca-4ad4-a32e-6925d72e9187.png">
This is because in the other functions you try to assign a new value to a  constant (todoArr). please kindly have a look at it and fix it